runtime/doc/usr_05.txt

```diff
@@ -250,121 +250,115 @@ The ":map" command (with no arguments) lists your current mappings.  At
 least the ones for Normal mode.  More about mappings in section |40.1|.
 
 ==============================================================================
-*05.4*	Adding a plugin					*add-plugin* *plugin*
-
-Vim's functionality can be extended by adding plugins.  A plugin is nothing
-more than a Vim script file that is loaded automatically when Vim starts.  You
-can add a plugin very easily by dropping it in your plugin directory.
-{not available when Vim was compiled without the |+eval| feature}
-
-There are two types of plugins:
-
-    global plugin: Used for all kinds of files
-  filetype plugin: Only used for a specific type of file
-
-The global plugins will be discussed first, then the filetype ones
-|add-filetype-plugin|.
-
-
-GLOBAL PLUGINS						*standard-plugin*
-
+*05.4*	Plugins					*add-plugin* *plugin*
+
+Vim's functionality can be extended by adding plugins.  A plugin is
+essentially a set of Vim script files that are loaded automatically when Vim
+starts or on some other event (like opening a file of a certain type).
+Plugins can be used to add support to more programming languages, extend Vim's
```

Comment:
Should read: 
Plugins can be used to add support *for* more programming languages,